Dockerfile.template

```diff
@@ -0,0 +1,96 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+FROM {{ BASE_IMAGE }}
+
+ARG spark_uid=185
+
+RUN set -ex && \
+    apt-get update && \
+    ln -s /lib /lib64 && \
+    apt install -y gnupg2 wget bash tini libc6 libpam-modules krb5-user libnss3 procps net-tools && \
```

Comment:
Consider using `-q` or even `-qq` for the `apt` commands.
The default output verbosity is usually too big.

Reply:
This patch would not change the dockerfile itself, if has problems, I prepare to do it in a separate PR. : )

I take a look on exsiting docker official image `-q` is not used: 
https://github.com/search?l=Dockerfile&p=3&q=org%3Adocker-library+apt&type=Code

I guess it might help to see the version/extra dependency packages.